.github/workflows/nightly-doc-build.yml

```diff
@@ -77,6 +77,7 @@ jobs:
 
       - name: "Deploy development documentation"
         uses: pyansys/actions/doc-deploy-dev@v2
+        if: matrix.image-tag == 'v23.1.0'
```

Comment:
@mkundu1 @seanpearsonuk We should only be deploying documentation to built against v22.2 right.  i.e. the doc here:  

https://dev.fluent.docs.pyansys.com/

should be only generated with v22.2 because that's all customers have right.

Reply:
So, I think this should say `if matrix.image-tag == 'v22.2.0'`

Reply:
I have added the check against 22.2 in #1199